pyccel/ast/core.py

```diff
@@ -2418,6 +2413,12 @@ def is_external(self):
         from a f77 module """
         return self._is_external
 
+    @is_external.setter
+    def is_external(self, is_external):
+        """
+        """
+        self._is_external = is_external
```

Comment:
Why not checking if the argument is a `bool` as was done originally in the `__init__`?

Reply:
Fixed [a2c04d1](https://github.com/pyccel/pyccel/pull/2318/commits/a2c04d1a6564ca26f3d910ee5955b277b5b97d2c)